src/features/fireWeather/components/NoonForecastDisplay.tsx

```diff
@@ -55,7 +54,7 @@ const DailyForecastDisplay = ({ values, testId, title }: Props) => {
               <TableRow>
                 <TableCell>Date</TableCell>
                 {values.map(v => (
-                  <TableCell key={v.datetime}>{formatDateInPDT(v.datetime)}</TableCell>
+                  <TableCell key={v.datetime}>{v.datetime.slice(0, 10)}</TableCell>
```

Comment:
why did you choose to move away from formatDateInPDT() ?

Reply:
I decided to only show month and day since they are always noon values, and I display `Weather forecast noon values (20:00 UTC):` in the title